src/main/java/com/google/sps/storage/StorageControllerInterface.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.storage;
+
+/**
+ * An interface for providing unified I/O to the storage component.
+ */
+public interface StorageControllerInterface {
```

Comment:
I'd rather drop the `Interface` suffix of this interface; it's a bit redundant as it's part of the type definition. I think StorageController would be an OK name. Now that brings up the follow-up question how to call the implementation class (that's now called StorageController). Since that implementation class is supposed to use Datastore to implement the functionality, I'd consider updating its name to something like DatastoreController or DatastoreStorageController.